File: apps/problems/trapping-rain-water/boilerplate-full/function.cpp
#include <iostream>
#include <vector>
#include <string>
#include <sstream>
#include <climits>



//##USER_CODE_HERE


int main() {
    int m_height;
std::cin >> m_height;
std::vector<int> height(m_height);

for(int i = 0; i < m_height; ++i) {
    std::cin >> height[i];
}
    Solution obj;
    int result = obj.trap(height);
    std::cout << result << std::endl;
    return 0;
}

File: apps/problems/trapping-rain-water/boilerplate-full/function.js

    

    //##USER_CODE_HERE##

const input = require('fs').readFileSync('/dev/problems/trapping-rain water/tests/inputs/##INPUT_FILE_INDEX##.txt', 'utf8').trim().split('\n').join(' ').split(' ');
const size_height = parseInt(input.shift() || "0");
        const height = input.splice(0, size_height).map(x => parseInt(x));
const result = trap(height);
console.log(result);
    